.github/workflows/build_v2.yml

```diff
@@ -0,0 +1,91 @@
+name: build-v2
+
+on: push
+
```

Comment:
Identical to build.yml (build file for v1), just with v2 working directory, and v2 in the node module cache